package/Dockerfile.dapper-base

```
@@ -51,8 +51,9 @@ RUN dnf -y distrosync --nodocs --setopt=install_weak_deps=False && \
     curl -Lo /usr/bin/kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-${ARCH}" && chmod a+x /usr/bin/kind && \
     curl -Lo /go/bin/subctl "https://github.com/submariner-io/submariner-operator/releases/download/${SUBCTL_VERSION}/subctl-${SUBCTL_VERSION}-linux-${ARCH}" && chmod a+x /go/bin/subctl && \
     GOFLAGS="" go get -v github.com/onsi/ginkgo/ginkgo && \
-    GOFLAGS="" go get -v golang.org/x/tools/cmd/goimports && \
-    upx /go/bin/{ginkgo,golangci-lint} /usr/bin/{containerd*,ctr,docker*,goimports,helm,kind,kube*,runc} /usr/lib/golang/pkg/tool/*/* && \
+    upx /go/bin/{ginkgo,golangci-lint} /usr/bin/{helm,kind,kubefedctl} && \
+    (rpm -ql containerd golang-bin golang-x-tools-goimports kubernetes-client moby-engine runc | \
+     grep -E '(/usr/bin|/usr/lib/golang/bin/|/usr/lib/golang/pkg/tool/.*/)' | xargs -r upx || :) && \
```

```suggestion
    find /usr/bin /go/bin /usr/lib/golang -type f -executable -size +5M | grep -v -e subctl | xargs upx && \
```

Might I suggest a simpler alternative? ;)

---

Indeed, simple is better, I’ve applied that (minus the `grep`).

---

I thought subctl is compressed already, hence the exclusion..